docs/pages/config/selection-policy.mdx

```diff
@@ -0,0 +1,192 @@
+import { Callout, Tabs, Tab } from 'nextra/components'
+
+### Selection Policy
+
+**Selection Policy** allows you to influence how upstreams are selected to serve (or not) traffic. A selection policy is defined at the network level and is responsible for returning a list of upstreams that must remain active. 
+
+The primary purpose of a selection policy is to define acceptable performance metrics and/or required conditions for selecting an upstream node.
+
+<Callout type='info'>
+  Selection policies can be configured to run per-method and network or per-network only. <br />
+</Callout>
+
+<Callout type='info'>
+   By default, a "default" selection policy is activated if at least one upstream is assigned to the "fallback" group. This default policy incorporates basic logic for error rates and block lag, which can be tuned via theese environment variables 
+   
+   * `ROUTING_POLICY_MAX_ERROR_RATE` (Default: `0.7`): Maximum allowed error rate.
+   * `ROUTING_POLICY_MAX_BLOCK_HEAD_LAG` (Default: `10`): Maximum allowed block head lag.
+   * `ROUTING_POLICY_MIN_HEALTHY_THRESHOLD` (Default: `1`): Minimum number of healthy upstreams that must be included in default group.
+   
+   These environment variables allow you to adjust the default logic without rewriting the policy function.
+</Callout>
+
+<Callout type='info'>
+  Selection policies are not executed per request, instead they run on an interval much like a healthcheck and update the available upstreams.
+</Callout>
+
+#### Use Cases
+
+- **Block Lag Management:** Disable upstreams that are lagging behind more than a specified number of blocks until they resync.
+- **Error Rate Handling:** Exclude upstreams exceeding a certain error rate and periodically check their status.
+- **Cost-Efficiency Optimization:** Prioritize "cheap" nodes and fallback to "fast" nodes only when necessary.
+
+#### Config
+
+<Tabs items={["yaml", "typescript"]} defaultIndex={0} storageKey="GlobalConfigTypeTabIndex">
+  <Tabs.Tab>
+```yaml filename="erpc.yaml"
+projects:
+  - id: main 
+
+    upstreams:
+      - endpoint: cheap-1.com
+      - endpoint: cheap-2.com
+      - endpoint: fast-1.com
+        # Each upstream can have an arbitrary group name which is used in metrics, as well as 
+        # useful when writing an eval function in selectionPolicy below.
+        group: fallback
+      - endpoint: fast-2.com
+        group: fallback
+
+    networks:
+      - architecture: evm
+        evm:
+          chainId: 1
+
+        # Determines when to include or exclude upstreams depending on their health and performance
+        selectionPolicy:
+          # Every 1 minute evaluate which upstreams must be included,
+          # based on the arbitrary logic (e.g., <90% error rate and <10 block lag):
+          evalInterval: 1m
+          
+          # Freeform TypeScript-based logic to select upstreams to be included by returning them:
+          evalFunction: |
+            (upstreams, method) => {
```

Comment:
One useful thing is a separate section to show the reference of upstreams/method types. You can copy from the generated.ts file and put comments on each field

e.g. errorRate is p90 of last X minutes (X is based on project.healthCheck.scoreMetricsWindowSize)